allensdk/brain_observatory/behavior/data_objects/eye_tracking/eye_tracking_table.py

```diff
@@ -236,6 +249,62 @@ def from_data_file(
 
         return EyeTrackingTable(eye_tracking=eye_tracking_data)
 
+    @classmethod
+    def _validate_frame_time_alignment(
+            cls,
+            frames: np.ndarray,
+            times: StimulusTimestamps,
+            is_metadata_frame_present: bool = False
+    ) -> Tuple[np.ndarray, StimulusTimestamps]:
+        """
+        Checks whether frames or timestamps need to be modified in order to be
```

Comment:
I think it is worth describing in the top docstring the overall logic of this validation process: what we do if number of frames greater than time stamps and when it is the other way around.